mobify/platform-scaffold
# Don't use merlins for package name

Changes some of the package descriptors for Android to enable the Generator to change the package.

 **JIRA**: https://mobify.atlassian.net/browse/PLATT-221

## Changes
- Change the package references from `merlins` to `scaffold`

## How to test-drive this PR
- `cd native`
- `npm run deps`
- Run the app in Android.
- The app should run and work 👍 

OR

- Deploy the build from buddybuild to a phone
- Run the app, should work 👍
